src/sqlite_back.cc

```diff
@@ -293,6 +293,7 @@ void SqliteBack::Bind(boost::spirit::hold_any v, DbTypes type, SqlStatement::Ptr
   CYCLUS_BINDVAL(MAP_STRING_STRING, std::map<std::string CYCLUS_COMMA std::string>);
   CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE, std::map<std::string CYCLUS_COMMA std::vector<double> >);
   CYCLUS_BINDVAL(MAP_STRING_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
+  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::pair<double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >);
```

Comment:
The diff was produced using the output of clang-format with the google style guide configuration.